handlers/accesscontrol/access-control.go

```diff
@@ -94,6 +96,13 @@ func (ac *AccessControlHandlersCollection) IsAuthorized(playbackID string, paylo
 		acReq.AccessKey = accessKey
 		cacheKey = "accessKey_" + accessKey
 	} else if jwt != "" {
+		for _, blocked := range ac.blockedJWTs {
+			if jwt == blocked {
+				glog.Errorf("blocking JWT. playbackId=%v jwt=%v", acReq.Stream, jwt)
+				return false, fmt.Errorf("JWT is blocked: %s", jwt)
```

Comment:
@mjh1 I _think_ we might not want to return an error here otherwise it will become a 500 to mist, which fails open. Not sure if someone else handles this error, but might be worth the double check

Reply:
Fixed here https://github.com/livepeer/catalyst-api/pull/1051